src/main/java/com/kainos/ea/backend/controllers/JobRoleController.java

```diff
@@ -25,5 +27,13 @@ public JobRoleController(JobRolesService jobRolesService) {
         return jobRolesService.getAllJobRolesSortedByCapability();
     }
 
-
+    @PostMapping("/add")
+    public ResponseEntity<?> addJobRole(@RequestBody JobRole jobRole){
```

Comment:
Should the <?> be there or should it read ResponseEntity<Object>?

Reply:
The wildcard needed to be there with the entry solution I've had, but now it is redundant so I will patch this up quick